tracks/go/exercises/raindrops/README.md

```diff
@@ -0,0 +1,65 @@
+# Raindrops
+
+This is the third core exercise in the go track to receice feedback from a mentor. It is less about learning new concepts but more about deepening the already acquired knowledge. To some the conversion of an `int` to `string` and the modula operator `%` are new.
```

Comment:
I believe this is typo, it's called modulo operator I believe

Reply:
Thanks! Will fix.